### ISSUE TITLE
Crash when scrolling articles

### ISSUE THREAD

**Issue by guillaume-tgl** [helpscout/beacon-android-sdk-sample#44]
* Beacon version: 0.3.0
* Android OS version: Android 8.0
* Device make/model: Samsung Galaxy S9

**Steps to reproduce**
I have only reproduced it within my app, not in the demo app so it might be specific to our Docs articles.
1. Open Beacon activity
2. Search "Export"
3. Scroll the list of articles

**Actual outcome**
It crashes.

**Expected outcome**
It shouldn't crash.

```
java.lang.IllegalArgumentException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkParameterIsNotNull, parameter receiver$0
        at com.helpscout.beacon.internal.core.extensions.StringExtensionsKt.cleanUpPreview(Unknown Source:2)
        at com.helpscout.beacon.internal.domain.search.b$a.a(Unknown Source:27)
        at com.helpscout.beacon.internal.domain.search.b$a.a(Unknown Source:2)
        at com.helpscout.beacon.internal.common.a.a.onBindViewHolder(Unknown Source:23)
        at android.support.v7.widget.RecyclerView$Adapter.onBindViewHolder(RecyclerView.java:6781)
        at android.support.v7.widget.RecyclerView$Adapter.bindViewHolder(RecyclerView.java:6823)
        at android.support.v7.widget.RecyclerView$Recycler.tryBindViewHolderByDeadline(RecyclerView.java:5752)
        at android.support.v7.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:6019)
        at android.support.v7.widget.GapWorker.prefetchPositionWithDeadline(GapWorker.java:286)
        at android.support.v7.widget.GapWorker.flushTaskWithDeadline(GapWorker.java:343)
        at android.support.v7.widget.GapWorker.flushTasksWithDeadline(GapWorker.java:359)
        at android.support.v7.widget.GapWorker.prefetch(GapWorker.java:366)
        at android.support.v7.widget.GapWorker.run(GapWorker.java:397)
        at android.os.Handler.handleCallback(Handler.java:789)
        at android.os.Handler.dispatchMessage(Handler.java:98)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6938)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:327)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1374)
```

**Comment by jotaemepereira** [comment]
@guillaume-tgl We are going to investigate this. Thanks for the clear report!

**Comment by scottyab** [comment]
Hey @guillaume-tgl  I've added handling for empty/null previews which seems to be the cause of the crash and it'll be included in our next release.

If you have an idea which article/doc is causing the crash and can give me the article Id or the content of the article I'll create test case using it to ensure the fix definiately works for you. 

 Thanks again for rasing this.

**Comment by guillaume-tgl** [comment]
@scottyab It seems like it's this one: https://android.help.thegrizzlylabs.com/article/135-how-to-export-a-scan-to-your-gallery-android

See how there is no preview in the search: https://android.help.thegrizzlylabs.com/search?query=export

**Comment by guillaume-tgl** [comment]
@scottyab By the way, do you know why no preview is displayed? :)

**Comment by scottyab** [comment]
Hey @guillaume-tgl We do some clean up on the preview to trim and convert any supported HTML tags but you should see a preview for each article. Can you share the Beacon Id you're using and I'll investigate further. Send to help@helpscout.com FAO Scott Alexander-Bown Thanks

**Comment by scottyab** [comment]
Fixed the crashing issue in 0.3.1 and moved the conversation about missing summary into Help Scout. Thanks for raising.
